pandas/tests/indexing/test_scalar.py

```diff
@@ -302,3 +303,29 @@ def test_iat_dont_wrap_object_datetimelike():
         assert result is ser2[1]
         assert isinstance(result, timedelta)
         assert not isinstance(result, Timedelta)
+
+
+def test_multiindex_series_loc_get():
+    series = pd.Series([1, 2], index=[[1, 2], [3, 4]])
+    assert series.loc[1, 3] == 1
+
+
+def test_multiindex_series_loc_set():
+    series = pd.Series([1, 2], index=[[1, 2], [3, 4]])
+    series.loc[1, 3] = 3
+    assert series.loc[1, 3] == 3
+
+
+def test_multiindex_series_at_get():
+    # GH 26989
+    # Series with multiindex at getter raises TypeError
+    series = pd.Series([1, 2], index=[[1, 2], [3, 4]])
+    assert series.at[1, 3] == 1
+
+
+def test_multiindex_series_at_set():
+    # GH 26989
+    # Series with multiindex at setter raises ValueError
```

Comment:
this comment refers to the behavior being fixed, not the desired behavior, right?  can you clarify that for future readers